Makefile.common

```diff
@@ -51,6 +52,31 @@ cacheartifact:
 	cekit-cache add ${ABSOLUTE_LOCAL_ARTIFACT_DIR} --md5 ${MD5SUM}
 	sed -i 's/- md5:.*/- md5: ${MD5SUM}/' modules/install-artifact/module.yaml 
 
+pushfrombrew:
+	$(eval SUBDIR_START_LINE=$(shell grep -n -m 1 $(CURDIR) $(CEKIT_OUTPUT_FILE) | cut -f1 -d:))
```

Comment:
What is the reason for using the `eval`?

Reply:
I am no expert in Makefiles.  But it expands things, to match Makefile syntax.  Without eval, it gives a syntax error, even if you use export etc.